inc/class-wpseo-replace-vars.php

```diff
@@ -1110,6 +1114,29 @@ private static function register_help_text( $type, $replace, $help_text = '' ) {
 		}
 	}
 
+	/**
+	 * Generate a list of replacevars based on the help texts.
+	 *
+	 * @return array List of replace vars.
+	 */
+	public function get_replacement_variables_list() {
+		self::setup_statics_once();
+
+		$replacement_variables = [];
+		foreach ( array_merge( self::$help_texts['basic'], self::$help_texts['advanced'] ) as $key => $value ) {
```

Comment:
I would choose for `array_filter`, with a separate method that filters the prefixed ones.

Feel free to let me help you implement this. I would love to pair program on that one.